src/service/SchemaServiceV2.ts

```diff
@@ -218,6 +218,16 @@ export class SchemaServiceV2 extends APIClient {
         return this.invokeApiWithErrorHandling(`/extensions/${name}`, 'PUT', extension, v2Header);
     };
 
+    /**
+     * Create extension.
+     * @param extension - The extension object, that should be created or updated.
+     * Throws http status code: CONFLICT(409) if an extension with the same name already exists
+     */
+    createExtension = async (extension: Extension) => {
+        return this.invokeApiWithErrorHandling(`/extensions`, 'POST', extension, v2Header);
```

Comment:
Is there a response ? 
```suggestion
        return this.invokeApiWithErrorHandling<???>(`/extensions`, 'POST', extension, v2Header);
```